src/Test/Utilities/Portable/CommonTestBase.CompilationVerifier.cs

```diff
@@ -107,9 +110,10 @@ public void Emit(string expectedOutput, IEnumerable<ResourceDescription> manifes
 
                     if (expectedOutput != null)
                     {
-                        testEnvironment.Execute(mainModuleName, expectedOutput);
+                        return testEnvironment.Execute(mainModuleName, expectedOutput, args);
                     }
                 }
+                return 1;
```

Comment:
>return 1; [](start = 16, length = 9)

Instead of not executing code in this case, we should probably pass expected result as nullable of int and force execution + comparison when it is not null.